aiida_quantumespresso/parsers/parse_raw/pw.py

```diff
@@ -366,6 +370,20 @@ def parse_stdout(stdout, input_parameters, parser_options=None, parsed_xml=None)
                         parsed_data['dipole' + units_suffix] = default_dipole_units
                         break
 
+            # saving the SCF convergence accuracy for each SCF cycle
+            # this is not always present in the block starting with 'iteration #'
+            # so the array might be of different size for each relax cycle
+            elif 'estimated scf accuracy' in line:
+                try:
+                    value = float(line.split()[-2])* ry_to_ev
+                    try:
+                        trajectory_data['scf_accuracy'].append(value)
```

Comment:
Can you please replace this old idiom with the `setdefault` one. Also please replace the other uses of the old idiom in this function if you will